apps/api/src/deployment/controllers/deployment/deployment.controller.ts

```diff
@@ -19,4 +28,33 @@ export class TopUpDeploymentsController {
   async cleanUpStaleDeployment(options: CleanUpStaleDeploymentsParams) {
     await this.staleDeploymentsCleanerService.cleanup(options);
   }
+
+  @Protected([{ action: "create", subject: "DeploymentSetting" }])
```

Comment:
blocking: this doesn't seem right. Why is it `DeploymentSetting`?

Reply:
At least this is what used currently to sign a create deployment message on user wallet. I'm not sure it is needed to add a new entity/action.